runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -221,4 +221,15 @@ default boolean claimSlime()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 16,
+			keyName = "swapClimb",
+			name = "Climb-Up",
+			description = "Swap climb with climb-up on staircases"
+	)
+	default boolean swapClimb()
+	{
+		return true;

```

Comment:
I'd set this swap option to `false` by default, otherwise it could be confusing for people trying to go downstairs.

Reply:
I think when you are going down it says climb-down and there wouldn't be an option to switch.

Reply:
Yeah, it won't affect stairs that only go down. But it will override the two-way `Climb` option on stairs in-between floors, which probably isn't desirable during normal gameplay.

Reply:
ok, yeah i see what you're saying, I changed that then.